examples/abstract-objects/tests/abstract_objects_test.bal

```diff
@@ -10,14 +9,14 @@ int counter = 0;
     functionName: "println"
 }
 public function mockPrint(any... s) {
-    outputs[counter] = string.convert(s[0]);
+    outputs[counter] = s[0].toString();
```

Comment:
@ramithaYO @keizer619 not introduced by this PR, but a significant number of tests seem to be comparing `.toString()` representations. Can we check the actual value instead? 

Basically add all to outputs here.
```suggestion
    outputs.push(s);
```

And when testing test against the actual values.

```ballerina
    test:assertEquals(outputs[0], "John Doe");
    test:assertEquals(outputs[1], 50); // `50` instead of `"50"`
```

If there are time constraints, I guess we can go ahead with these changes for the time-being. But let's create an issue to track improving these. WDYT?